### ISSUE TITLE
Parts of Logging-in for Non-Google Accounts in Staging are not Working

### ISSUE THREAD

**Issue by KatrinaJewel** [rtercias/foreign-field#357]
There are two issues that might be related:
1. Logging in with a non-Gmail email - 
Expected behavior:
- be able to set the password and login
Actual behavior:
- password was set > "You are not authorized" is displayed

2. Logging in via phone number while the account is inactive - 
Expected behavior:
- Be able to login > "You are not authorized" message
Actual behavior:
- User is able to login but it just tries to load. "You are not authorized" is not displayed

**Comment by rtercias** [comment]
I made some updates on the email/password sign-in flow. Fixed quite a few bugs, including initial setup for new account. Please test. Thanks!

**Comment by rtercias** [comment]
I'll take a look at the phone bug next.

**Comment by rtercias** [comment]
I've made some fixes related to the phone login flow. Also did some refactoring of the logout process. All items in this issue should be fixed now in staging.

**Comment by KatrinaJewel** [comment]
The email and the phone issues are fixed now. 

When logging out of a google account it says logged out but when logging back in it no longer asks which account or for a password, it automatically opens in the last used google account. Is this new expected behavior?

**Comment by rtercias** [comment]
Good question... this was the original intent of Google single sign-on, to automatically sign you in as long as your Google account is active. I had added an optional step to "log out of Google account", but that was to help others to not leave their account open. Nowadays, that's mostly unnecessary, so I've removed that Google log out button.

If there's a need for anyone to log out of their Google account, browsers like Chrome now includes a "sign out" button.
